app/services/fastboot.js

```diff
@@ -4,6 +4,22 @@ import Ember from "ember";
 const { deprecate, computed, get } = Ember;
 const { alias, deprecatingAlias } = computed;
 
+const ResponseObject = Ember.Object.extend({
```

Comment:
There was a very specific reason I did this for "response", and that was so I could have a computed property for host, while having direct access to the others. Alias for response in the service is fine